reference/ironclaw-flow-atlas.html

```
@@ -0,0 +1,205 @@
+        <table class="compare">
+          <thead>
+            <tr>
+              <th>User prompt</th>
+              <th>Flow skeleton</th>
+              <th>Diagnostic question</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>"Explain this Rust module."</td>
+              <td>Product -> thread -> turn/run -> runner -> loop -> model -> final reply.</td>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)